how to build your own website/blog.md

```diff
@@ -0,0 +1,253 @@
+# How to Build Your Own Website - Introduction to HTML, CSS, and JavaScript
+
+The world of web development and design is growing, and creating a website, whether it would be to display a design portfolio or to track personal progress in everyday life, has become a popular method to establish an online presence or to track personal progress. As the selection of website builders and frameworks continue to expand, however, it can seem daunting and overwhelming when you do not know where or how to start.
```

Comment:
You repeat yourself here a bit with the "personal progress" phrase. Please go through the entire blog and thoroughly proofread it, to catch any awkward wording or grammar mistakes!